pom.xml

```diff
@@ -96,13 +96,13 @@
         <vertx.version>4.2.4</vertx.version>
         <vertx-junit5.version>4.2.4</vertx-junit5.version>
         <vertx.kafka.client>4.2.4</vertx.kafka.client>
-        <log4j.version>2.17.1</log4j.version>
+        <log4j.version>2.17.2</log4j.version>
         <hamcrest.version>2.2</hamcrest.version>
         <valid4j.version>1.1</valid4j.version>
-        <fasterxml.jackson-core.version>2.11.3</fasterxml.jackson-core.version>
-        <fasterxml.jackson-databind.version>2.11.3</fasterxml.jackson-databind.version>
-        <fasterxml.jackson-dataformat.version>2.10.5</fasterxml.jackson-dataformat.version>
-        <fasterxml.jackson-annotations.version>2.11.3</fasterxml.jackson-annotations.version>
+        <fasterxml.jackson-core.version>2.12.6</fasterxml.jackson-core.version>
+        <fasterxml.jackson-databind.version>2.12.6</fasterxml.jackson-databind.version>
+        <fasterxml.jackson-dataformat.version>2.12.6</fasterxml.jackson-dataformat.version>
+        <fasterxml.jackson-annotations.version>2.12.6</fasterxml.jackson-annotations.version>
```

Comment:
Lets see if this works ... Jackson updates are often more complicated than just bumping the versions, especially if you change the minor version.

Reply:
It looks like this generates many changes to the different files:
```
ERROR: Uncommitted changes in derived resources:
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/040-Crd-kafka.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/041-Crd-kafkaconnect.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/042-Crd-strimzipodset.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/043-Crd-kafkatopic.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/044-Crd-kafkauser.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/045-Crd-kafkamirrormaker.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/046-Crd-kafkabridge.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/047-Crd-kafkaconnector.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/048-Crd-kafkamirrormaker2.yaml
M	packaging/helm-charts/helm3/strimzi-kafka-operator/crds/049-Crd-kafkarebalance.yaml
M	packaging/install/cluster-operator/040-Crd-kafka.yaml
M	packaging/install/cluster-operator/041-Crd-kafkaconnect.yaml
M	packaging/install/cluster-operator/042-Crd-strimzipodset.yaml
M	packaging/install/cluster-operator/043-Crd-kafkatopic.yaml
M	packaging/install/cluster-operator/044-Crd-kafkauser.yaml
M	packaging/install/cluster-operator/045-Crd-kafkamirrormaker.yaml
M	packaging/install/cluster-operator/046-Crd-kafkabridge.yaml
M	packaging/install/cluster-operator/047-Crd-kafkaconnector.yaml
M	packaging/install/cluster-operator/048-Crd-kafkamirrormaker2.yaml
M	packaging/install/cluster-operator/049-Crd-kafkarebalance.yaml
M	packaging/install/topic-operator/04-Crd-kafkatopic.yaml
M	packaging/install/user-operator/04-Crd-kafkauser.yaml
```

I don't think the Jackson issue really affects Strimzi anyway, so I think you should probably remove it from this PR.